eddy/core/items/common.py

```diff
@@ -380,12 +382,12 @@ def focusOutEvent(self, focusEvent):
 
             self.focusInData = None
             self.setSelectedText(False)
-            self.setAlignment(self.alignment())
+            self.wrapLabel()
             self.setTextInteractionFlags(QtCore.Qt.NoTextInteraction)
             self.diagram.setMode(DiagramMode.Idle)
             self.diagram.sgnUpdated.emit()
 
-        self.setAlignment(self.alignment())
+        self.wrapLabel()
```

Comment:
Come mai è necessario fare il wrap del label durante la perdita del focus? Non credo questo evento sia più rilevante in quanto l'editing dei label non avviene più in maniera interattiva. Mi sfugge qualcosa?

Reply:
L'ho messo giusto perché il `setAlignment` è stato inglobato nella `wrapLabel` e quindi l'ho sostituito in tutte le occorrenze, ma se pensi che non sia necessario posso toglierlo

Reply:
Ok allora avevo intuito bene, praticamente agganci la nuova funzione dove prima veniva settato l'allineamento. Per il momento credo possiamo lasciarla così, se ci viene in mente una versione migliore poi la cambiamo.